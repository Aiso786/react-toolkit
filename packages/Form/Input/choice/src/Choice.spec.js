import React from 'react';
import renderer from 'react-test-renderer';
import ChoiceInput from './ChoiceInput';

describe('<ChoiceInput>', () => {
  it('renders ChoiceInput correctly', () => {
    const tree = renderer
      .create(
        <ChoiceInput
          label="Image *"
          id="iddateinput"
          name="placeImage"
          options={[{ id: 'iddateinput', value: 'sample' }]}
          onChange={() => {}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
