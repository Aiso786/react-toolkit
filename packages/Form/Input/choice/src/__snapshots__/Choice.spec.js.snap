// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChoiceInput> renders ChoiceInput correctly 1`] = `
<div
  className="row af-form__group"
>
  <div
    className="col-md-2"
  >
    <label
      className="af-form__group-label"
      htmlFor=""
    >
      Image *
    </label>
  </div>
  <div
    className="col-md-10"
  >
    <div
      className="af-form__radio-custom"
    >
      <input
        autoFocus={null}
        checked={false}
        className="af-form__input-radio"
        componentClassName="af-form__radio-custom"
        disabled={false}
        id="iddateinput"
        name="placeImage"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={false}
        tabIndex={null}
        type="radio"
        value="sample"
      />
      <label
        className="af-form__label"
        htmlFor="iddateinput"
      >
        <span
          className="af-form__description"
        />
      </label>
    </div>
  </div>
</div>
`;
